src/lib/components/works_for_you/notification.tsx

```diff
@@ -13,7 +13,9 @@ interface Props extends RelayProps {}
 
 export class Notification extends React.Component<Props, any> {
   handleArtistTap() {
-    const artistHref = this.props.notification.artworks[0].artists[0].href
+    // Special notifications will pass down an artistHref. Otherwise, grab it from the artworks
```

Comment:
I suppose if i decoupled the header from the notification, I could make a new `specialNotification` component that uses the notification header and the artworks grid and that might be more in line with the React way of doing things. But it would also be slightly more redundant.